platform/darwin/src/MGLOfflineStorage.h

```diff
@@ -238,6 +238,31 @@ typedef void (^MGLOfflinePackRemovalCompletionHandler)(NSError * _Nullable error
  */
 @property (nonatomic, readonly) unsigned long long countOfBytesCompleted;
 
+/**
+ Sets a URL template for the given scheme.
```

Comment:
Not sure how `Any URLs encountered in styles and subsequently loaded sources` could be any clearer